js/main.js

```diff
@@ -462,13 +462,18 @@ function fixMapHeight() {
 $(window).on('resize', fixMapHeight);
 
 
-$(window).on('hashchange',function() {
-    // Don't create a new history state, because the hash change already did
-    setCity(getPathCity(), false);
-});
-
+function updateOldUrlScheme(){
+    // we used to pass the city after the `#` in the url.
```

Comment:
Please move the documentation above the function as we did for most of the other functions.